py3status/modules/spaceapi.py

```diff
@@ -57,10 +52,7 @@ def check(self):
                 self.closed_color = ''
 
             # grab json file
-            json_file = urllib.request.urlopen(self.url)
-            reader = codecs.getreader("utf-8")
-            data = json.load(reader(json_file))
-            json_file.close()
+            data = self.py3.get_json_data(self.url)
```

Comment:
It was being implicitly checked, since the `try` block was catching every exception (which I am not a fan of).  `None` will throw an `TypeError` or `AttributeError` when accessing the attributes in the following lines.

I had other improvements, but I removed them because they would deviate from the previous behavior (an empty string for `full_text`).  I will address that when implementing the format update.